### DISCUSSION TITLE
Trying to upgrade to Pro account but payment is automatically refunded back in few seconds.

### DISCUSSION THREAD

**Discussion by covert-oddity** [vercel/community#1635] [Help]
### Summary

Hi, I am trying to upgrade my hobby plan, the amount gets charged from my card, but immediately refunded and now my deployments are blocked. Please help. As I am not a paid customer now, I can't even reach the customer support. Please I need urgent help.

**Comment by amyegan** [comment]
Hi @covert-oddity. Can you open a new case under the "Billing" problem area and let me know the case number? That way we can solve the payment bug in private and keep your info safe 🙏
